scipp-imaging-2D.ipynb

```diff
@@ -0,0 +1,8350 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to start\n",
+    "\n",
+    "Before starting you must ensure that `scipp` and `mantid` are on your `PYTHONPATH`.\n",
+    "\n",
+    "For `scipp` and `mantid` follow instructions at: https://scipp.readthedocs.io/en/latest/getting-started/installation.html.\n",
+    "\n",
+    "Additionally, the other package dependencies are `fabio` and `astropy`: `conda install fabio astropy`\n",
+    "\n",
+    "This notebook follows the steps and uses the parameters from this reduction script https://git.esss.dk/testbeamline/gp2/blob/master/reduce.py but have them converted using equivalent `scipp` functionality.\n",
+    "\n",
+    "Before starting you need to configure:\n",
+    "\n",
+    "- The experiment directory in the `experiment_dir` variable - this "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "# Specify where the data can be found\n",
+    "experiment_dir = \"/home/dtasev/dev/scipp_imaging/RAL_Mantid_Sep2018/\"\n",
+    "\n",
+    "# let's get the process started:\n",
+    "tofs_path = os.path.join(experiment_dir, 'data_GP2', 'metadata', 'GP2_BCC_time_values.txt')\n",
+    "sample_path = os.path.join(experiment_dir, 'data_GP2', 'Timeslices WFM BBC Steel')\n",
+    "ob_path = os.path.join(experiment_dir, 'data_GP2', 'Timeslices WFM Open Beam')\n",
+    "\n",
+    "# defining grouping of 2D detector pixels\n",
+    "grouping_number = 3\n",
+    "nx_target = grouping_number\n",
+    "ny_target = grouping_number\n",
+    "\n",
+    "frame_parameters = [(15167,23563,64),\n",
```

Comment:
Could you add more information here. This is more public facing that the scripts you originally worked from. What are these values, what units. Maybe could immediately be encoded as scipp variables as a self-documenting approach?